dev-libs/sblim-sfcc/sblim-sfcc-2.2.8.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Small Footprint CIM Client Library"
+HOMEPAGE="https://sourceforge.net/apps/mediawiki/sblim"
+SRC_URI="mirror://sourceforge/project/sblim/${PN}/${P}.tar.bz2"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	net-misc/curl[ssl]
+	"
+RDEPEND="${DEPEND}"
+DOCS=()
+
+src_configure() {
+	econf --enable-http-chunking \
+	--disable-static
+}
+
+src_install() {
```

Comment:
This doesn't install any files…

Reply:
Then were should I remove the files? It needs to be after building right?

Reply:
The problem is that portage now does not know how to install the package, previously it used `src_install()` declared by default (so developers did not have to declare it manually), now you declared `src_install()` manually adding .la files removal and did not tell portage what to do with instllation...

Reply:
I see. If I remember correctly there was a way to call the normal src_install() and do do special changes. Was it using 'default' as the first call in the function?

Reply:
Exactly :). And as far as I can judge the ebuild still installs static libs (I just tried to install this), so please double check that.